telethon/tl/custom/message.py

```diff
@@ -617,19 +617,21 @@ def get_entities_text(self, cls=None):
                 >>> for _, inner_text in m.get_entities_text(MessageEntityCode):
                 >>>     print(inner_text)
         """
+        entities_list = self.original_message.entities
```

Comment:
This should be moved after the `if`, since it's not necessary `if not self.original_message.entities:` fails.